examples/light-switch-app/silabs/SiWx917/BUILD.gn

```diff
@@ -67,10 +67,10 @@ declare_args() {
 
 # Sanity check
```

Comment:
The specific SiWx917 folders are use for the SiWx917 WiFi platform only right? Correct me if i am wrong but there is no plan to support using another wifi platform ars rcp/ncp (wf200 or rs9116) or even openthread build for that matter. 

Why do we continue to have all those if use_SiWx917, if use_wf200, if wifi, if openthread when this is a pure wifi soc platform.

There is no conditions to have. If you use examples/<Whaever-app>/silabs/SiWx917/BUILD.gn It is going to be a WiFi app with SiWx917

Reply:
@jmartinez-silabs  yeah true, cleanup will be a part of a different PR on the SiWx917 SOC. 
removing of wf200 and openthread from the SiWx917 platform gn system will be taken in the cleanup in a different PR